collectors/osv/collectors.py

```diff
@@ -166,11 +168,14 @@ def save_snippet(
             return 0, 0
 
         if not cve_ids:
-            _, created = Snippet.objects.update_or_create(
+            # This keeps the structure consistent and ease snippets filtering without cve_id
+            content["cve_id"] = None
+            snippet, created = Snippet.objects.update_or_create(
                 source=Snippet.Source.OSV,
                 external_id=osv_id,
                 defaults={"content": content},
             )
+            snippet.convert_snippet_to_flaw()
```

Comment:
This seems to be creating new flaws also on snippet update.

Reply:
Thanks, I will check it.